apps/rule-manager/app/controllers/RulesController.scala

```diff
@@ -190,4 +194,16 @@ class RulesController(
         }
     }
   }
+
+  def testWithBlock(id: Int) = APIAuthAction[JsValue](parse.json).async { implicit request =>
+    request.body.validate[Document].asEither match {
+      case Right(document) =>
+        val rule = DbRuleDraft.find(id).get
```

Comment:
At one point in the commit history we handled the rule's optionality, but it seems we've lost this?

Reply:
Thanks for spotting! Added in 4cde00e17460cfc26e3ad56825341ab42bf25621.